lib/configurators/aws/aws_test.go

```diff
@@ -2333,6 +2334,13 @@ type SSMMock struct {
 func (m *SSMMock) CreateDocumentWithContext(ctx aws.Context, input *ssm.CreateDocumentInput, opts ...request.Option) (*ssm.CreateDocumentOutput, error) {
 
 	m.t.Helper()
+
+	// UUID's are unpredictable, so we remove them from the content
```

Comment:
Same remarks as https://github.com/gravitational/teleport/pull/35364#discussion_r1416157843:

> Would it make sense to push this to the test, instead of having the mock edit the document? The test might not need to compare entire docs, or it could craft the wanted doc more precisely.